wordpress-mobile/WordPress-Android
# Lower max saved log files and update utils

Fixes #13271 

We're having quite a lot of crashes when there is no more space left on the device. Is there a reason @loremattei why we need this many (30) last log files stored on the device? I feel like 5 should be more than enough. Let me know what you think! 

To test:
- Make sure you don't get this `Log.e("LogFileWriter", "Writing log failed: ...")` message on app start.

PR submission checklist:
